frontend/src/metabase/admin/databases/components/DatabaseHelpSidePanel/DatabaseHelpSidePanel.tsx

```diff
@@ -0,0 +1,117 @@
+import { useDisclosure } from "@mantine/hooks";
+import { Link } from "react-router";
+import { c, t } from "ttag";
+
+import { NewUserModal } from "metabase/admin/people/containers/NewUserModal";
+import { useSetting } from "metabase/common/hooks";
+import { getHelpUrl } from "metabase/common/utils/help-url";
+import CS from "metabase/css/core/index.css";
+import { getEngines } from "metabase/databases/selectors";
+import { useSelector } from "metabase/lib/redux";
+import { getDocsUrl, getIsPaidPlan } from "metabase/selectors/settings";
+import { getUserIsAdmin } from "metabase/selectors/user";
+import { getShowMetabaseLinks } from "metabase/selectors/whitelabel";
+import {
+  ActionIcon,
+  Box,
+  Button,
+  Divider,
+  Flex,
+  Icon,
+  Title,
+} from "metabase/ui";
+import type { EngineKey } from "metabase-types/api";
+
+import {
+  ENGINE_DOC_MAP,
+  EmbeddedEngineDocContent,
+} from "./EmbeddedEngineDocContent";
+
+interface Props {
+  engineKey: EngineKey;
+  onClose: VoidFunction;
+}
+
+export const DatabaseHelpSidePanel = ({ engineKey, onClose }: Props) => {
+  const engines = useSelector(getEngines);
+  const docsUrl = useSelector((state) =>
+    getDocsUrl(state, {
+      page: `databases/connections/${ENGINE_DOC_MAP[engineKey]}`,
+    }),
+  );
```

Comment:
There is a useful hook called `useDocsUrl` that can help you here. It already contains `showMetabaseLinks`.